tests/testthat/files/device.R

```diff
@@ -1,34 +1,34 @@
-#* @png
+#* @serializer png
```

Comment:
Will these changes break `@png`, etc? If so, let's make sure to mention it in a breaking changes section of the news

Reply:
It will not break them. I just don't want to encourage it. 

Should we throw a deprecated warning when parsing?

Reply:
Ah, ok, yea, a warning or message seems like a good idea

Reply:
Same for `@json` and `@html`? 🤞🤞

Reply:
Doing for them as well.